cypress/e2e/dashboard/chapters/[chapterId]/edit.cy.ts

```diff
@@ -73,4 +73,28 @@ describe('chapter edit dashboard', () => {
       cy.contains(chapterData.name);
     });
   });
+
+  it('deny admins to delete a chapter', () => {
+    const chapterId = 1;
+    cy.login(Cypress.env('JWT_ADMIN_USER'));
+
+    cy.updateChapter(chapterId, chapterData).then((response) => {
+      expectToBeRejected(response);
+
+      cy.visit(`/dashboard/chapters/${chapterId}`);
+      cy.findByRole('button', { name: 'Delete Chapter' })
+        .click()
+        .findByRole('button', { name: 'Are you sure?' })
+        .click()
+        .findByRole('button', { name: 'Delete' })
+        .click();
```

Comment:
```suggestion
```
There's no need to lock down a broken UI with a test.  As well as the server rejecting the request, the user should not be able to make the request through the UI.